### DISCUSSION TITLE
Cool online tool that might be useful

### DISCUSSION THREAD

**Discussion by Jdgilby** [Riverscapes/riverscapes-tools#824] [General]
@MattReimer @KellyMWhitehead @philipbaileynar I ran into processing problems when trying to fix the national dataset and ended up running across Mapshaper.org. I found it extremely useful! [check it out](https://youtu.be/iuQ8umhILyg)

**Comment by MattReimer** [comment]
> @MattReimer @KellyMWhitehead @philipbaileynar I ran into processing problems when trying to fix the national dataset and ended up running across Mapshaper.org. I found it extremely useful! [check it out](https://youtu.be/iuQ8umhILyg)

This is an EXCELLENT find! We should all bookmark this.
